src/components/Timelog/TimeEntry.jsx

```diff
@@ -84,6 +84,20 @@ const TimeEntry = (props) => {
     //permission to delete any time entry on their own time logs tab
     dispatch(hasPermission('deleteTimeEntry')) ||
     //default permission: delete own sameday tangible entry
+    (!isTangible && isAuthUser && isSameDay);
+
+  const canEdit =
```

Comment:
This variable name was already declared.
Maybe attempt to comment out previous declaration.